dbms/src/IO/CompactContext.cpp

```diff
@@ -38,8 +38,8 @@ extern const int UNKNOWN_FORMAT;
 extern const int FORMAT_VERSION_TOO_OLD;
 } // namespace ErrorCodes
 
-CompactWriteCtx::CompactWriteCtx(std::string compact_path_, size_t buffer_size)
-    : file_name(compact_path_)
+CompactWriteCtx::CompactWriteCtx(std::string part_path, size_t buffer_size)
+    : file_name(part_path)
```

Comment:
Why not use `std::move(part_path)` here?

Reply:
Is this also deleted?

Reply:
yes